OxfordBodleian/Juel-Jensen/BDLaethf34.xml

```diff
@@ -150,38 +699,199 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    Rubrication was possibly executed in a different hand.
+                                </seg>
+                                <list type="abbreviations"> 
+                                    <item>
+                                        <abbr>ቅ፡</abbr> for <expan>ቅዱስ፡</expan> (e.g. on <locus target="#2va #9rb #31ra"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ገ፡ ለ፡</abbr> for <expan> ገነይነ፡ ለኪ፡</expan> (e.g. on <locus target="#10ra"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ቅ፡ እ፡ ጸ፡ አ፡</abbr> for <expan>ቅዱስ፡ እግዚአብሔር፡ ጸባዖት፡ አምላክነ፡</expan> (<locus from="31ra" to="31va"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ሃ፡ ሉ፡</abbr> for <expan>ሃሌ፡ ሉያ፡</expan> (<locus from="32va" to="33ra"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>ይ፡ ሕ፡</abbr> for <expan>ይቤ፡ ሕዝብ፡</expan> (e.g. on <locus target="#33ra #37ra"/>)
+                                    </item>
+                                   <item>
+                                       <abbr>ይ፡ ካ፡</abbr> for <expan>ይቤ፡ ካህን፡</expan> (e.g. on <locus target="#33vb #36vb"/>)
+                                    </item>
+                                       <item>
+                                           <abbr>ይ፡ ዲ፡</abbr> for <expan>ይቤ፡ ዲያቆን፡</expan> (e.g. on <locus target="#33vb #36vb"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>በ፡ ሰ፡</abbr> or <abbr>በሰ፡ ሰ፡</abbr> for <expan>በሰጊድ፡ ሰላም፡</expan> (<locus from="37vb" to="41vb"/>)
+                                    </item>
+                                </list>
+                        
                             </handNote>
                         </handDesc>
                         
-                       <!-- <decoDesc>
-                         
-                        </decoDesc>-->
+               <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="MagicFormula">. The text is written in a poor hand in dark brownish ink.</desc>  
+                                    <locus target="#1r"/>
+                                    <q xml:lang="gez">ክርክምና፡ ክምና፡ ኵልኩልና፡ ቀርቃና፡ ሜሎስ፡ <del rend="effaced" unit="chars" quantity="1"/>ዴን፡ ቆቁና፡ ፌወቆቁሬሁ፡ ቁሽ፡ 
+                                        ቃለ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ዘተሰወረኒ፡ ክሰት፡ ሊተ፡ አባ፡</q>
+                                    </item>
+                                <item xml:id="a2">
+                                    <desc>Title of the book and name of the owner.
```

Comment:
It's all fine, just a general remark: When choosing what to mark up, it would be a bit preferable to mark up dates, names etc. in the quoted text instead of the description (whihc is also fine though). It would also be good to add a `<date>` element to the item (I also forget this a lot).

Reply:
In this case the dating (`<date>4 March 1864</date>`) is embedded in the description in a discursive way. Do you mean `<date>` should be provided in a more schematic way?

Reply:
It's probably fine. But it would be nice to provide the date in the markup (`<date when="1864-03-04">`), so that later it can be better filtered.